frontend/packages/topology/integration-tests/features/export-application/export-of-applicaton.feature

```diff
@@ -1,83 +1,69 @@
-@topology @broken-test
+@topology
 Feature: Export of application
               As a user, I have an unmanaged application which I want to export. I'd like to be able to later add that code to git or some shared location so that I can share with others, or import into a new cluster or same cluster but different project, or be able to apply updates to an existing application.
 
 
 
-    #The operator mentioned below needs to be installed beforehand due to the issue in its installation
         Background:
-            # Given user has installed Gitops primer Operator
+             Given user has installed Gitops primer Operator
               And user is at developer perspective
               And user has created or selected namespace "aut-export-application"
 
-
-#marking below test as broken due to intermittent issue of not getting toast notification after export is complete
-        @smoke @broken-test
+        @regression @odc-6738
         Scenario: Export application button in topology: EA-02-TC01
-            Given user has created a deployment workload "nodejs-ex-git-1"
-             When user navigates to Topology page
-              And user clicks on Export Application button
+            Given user has created "nodejs-ex-git-1" workload in "nodejs-ex-git-app" application
+             When user clicks on Export Application button
+              And user clicks on Ok button on Export Application modal to start the export
              Then user can see a toast message saying "Export of resources in aut-export-application has started."
-              And user can see a toast message saying "All the resources are exported successfully from aut-export-application. Click below to download it." and close it
+              And user can see a toast message saying "All the resources are exported successfully from aut-export-application. Click below to download it." with download option and close button
               And user can see primer job created in topology
 
 
         @regression @odc-6296
         Scenario: Export Application modal when export application is in progress: EA-02-TC02
-            Given user has created a deployment workload "nodejs-ex-git-2"
-              And user is at Topology page
+            Given user has created "nodejs-ex-git-2" workload in "nodejs-ex-git-app" application
              When user clicks on Export Application button
-              And user clicks on Export Application button again
+              And user clicks on Ok button on Export Application modal to start the export
+              And user clicks on Export Application button
              Then user can see "View logs" link, "Cancel Export", "Restart Export", and "Ok" button
```

Comment:
```suggestion
             Then user can see "View logs" link, "Cancel Export", "Restart Export", and "Ok" button
              And user can see a toast message saying "All the resources are exported successfully from aut-export-application. Click below to download it." with download option and close button
              And user can see primer job created in topology
```

Reply:
Here I am only verifying the presence of the mentioned buttons and not clicking on "Ok"/"Restart"

Reply:
Yes, but we need to wait for the export to complete else it will affect other tests

Reply:
umm but my next automated test case is to verify the restart action